contrib/resources/translations/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(FILES
+
+    br.lng
+    de.lng
+    es.lng
+    fr.lng
+    it.lng
+    nl.lng
+    pl.lng
+    ru.lng
+)
```

Comment:
Yeah, so all the rest of the crap can go into `contrib/translation-scripts`.

Reply:
We are also omitting `en.lng` here. I would like to get rid of the scripts in the future and just use the gettext format, but we are not there yet.